iml-agent/Cargo.toml

```diff
@@ -40,6 +40,7 @@ parking_lot = "0.9"
 iml-wire-types = { path = "../iml-wire-types", version = "0.2" }
 liblustreapi = { path = "../liblustreapi", version = "0.1" }
 iml-fs = { path = "../iml-fs", version = "0.1.0" }
+regex = "1.3.1"
```

Comment:
We can shrink the binary size a bit by not including all features:

```
[dependencies.regex]
version = "1.3"
default-features = false
features = ["std"]
```

Looking at the usage, I think this should be ok.

Reply:
Done